Doc/c-api/structures.rst

```diff
@@ -294,3 +294,52 @@ definition with the same method name.
    read-only access.  Using :c:macro:`T_STRING` for :attr:`type` implies
    :c:macro:`READONLY`.  Only :c:macro:`T_OBJECT` and :c:macro:`T_OBJECT_EX`
    members can be deleted.  (They are set to *NULL*).
+
+
+.. c:type:: PyGetSetDef
+
+   Structure to define property-like access for a type. See also description of
+   the :c:member:`PyTypeObject.tp_getset` slot.
+
+   +-------------+------------------+-----------------------------------+
+   | Field       | C Type           | Meaning                           |
+   +=============+==================+===================================+
+   | name        | char \*          | attribute name.                   |
+   +-------------+------------------+-----------------------------------+
+   | get         | :c:type:`getter` | C Function to get the attribute   |
+   +-------------+------------------+-----------------------------------+
+   | set         | :c:type:`setter` | Optional C function to set or     |
+   |             |                  | delete the attribute.  If omitted |
+   |             |                  | the attribute is readonly.        |
+   +-------------+------------------+-----------------------------------+
+   | doc         | char \*          | Optional docstring                |
+   +-------------+------------------+-----------------------------------+
+   | closure     | void \*          | Optional function pointer         |
+   |             |                  | providing additional data for     |
+   |             |                  | getter and setter.                |
+   +-------------+------------------+-----------------------------------+
+
+
+.. c:type:: getter
```

Comment:
I have opened [a thread on Python-Dev](https://mail.python.org/pipermail/python-dev/2017-March/147686.html). If the decision will be to add `Py`-prefixed aliases, they can be documented as common types. Otherwise I would prefer to not document them explicitly (as well as a number of other typedefs in [`typeobj.rst`](https://docs.python.org/3/c-api/typeobj.html)).